modules/mod_ginger_base/translations/nl.po

```diff
@@ -732,6 +775,13 @@ msgstr "resultaten"
 msgid "search"
 msgstr "zoeken"
 
+#~ msgid "More than 30000 results"
```

Comment:
Staan deze comments hier nog ergens voor?

Reply:
Dat zijn oude strings - die houdt de PO editor vaak nog even in de file.